core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeSorterSpillReader.java

```diff
@@ -59,21 +60,22 @@ public UnsafeSorterSpillReader(
       File file,
       BlockId blockId) throws IOException {
     assert (file.length() > 0);
+    final ConfigEntry<Object> bufferSizeConfigEntry =
+        package$.MODULE$.UNSAFE_SORTER_SPILL_READER_BUFFER_SIZE();
```

Comment:
Oh, I didn't consider that this is Java file.

Reply:
This can't be `ConfigEntry<Long>` because the Scala side has the generic type as a `long`? It seems like it ends up a `Long` anyway as you're able to `.get()` and cast it to `long`.

Reply:
Yes. It looks like that to me, too.